VideoWeb/VideoWeb/Extensions/ConfigureAuthSchemeExtensions.cs

```diff
@@ -0,0 +1,144 @@
+                }).AddPolicyScheme(JwtBearerDefaults.AuthenticationScheme, "Handler", options =>
+                    options.ForwardDefaultSelector = context =>
+                        context.Request.Path.StartsWithSegments("/callback")
+                            ? "Callback" : "Default")
+                .AddJwtBearer("Default", options =>
+                {
+                    options.Authority = $"{securitySettings.Authority}{securitySettings.TenantId}";
+                    options.TokenValidationParameters.ValidateLifetime = true;
+                    options.Audience = securitySettings.ClientId;
+                    options.TokenValidationParameters.ClockSkew = TimeSpan.Zero;
+                }).AddJwtBearer("EventHubUser", options =>
+                {
+                    options.Events = new JwtBearerEvents
+                    {
+                        OnMessageReceived = context =>
+                        {
+                            var accessToken = context.Request.Query["access_token"];
+                            if (string.IsNullOrEmpty(accessToken)) return Task.CompletedTask;
+
+                            var path = context.HttpContext.Request.Path;
+                            if (path.StartsWithSegments(eventhubPath))
+                            {
+                                context.Token = accessToken;
+                            }
+
+                            return Task.CompletedTask;
+                        }
+                    };
+                    options.Authority = $"{securitySettings.Authority}{securitySettings.TenantId}";
+                    options.TokenValidationParameters = new TokenValidationParameters()
+                    {
+                        ClockSkew = TimeSpan.Zero,
+                        ValidateLifetime = true,
+                        ValidAudience = securitySettings.ClientId
+                    };
+                }).AddJwtBearer("Callback", options =>
+                {
+                    options.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidateIssuer = false,
+                        ValidateAudience = false,
+                        IssuerSigningKey = new SymmetricSecurityKey(securityKey)
+                    };
+                });
+
+            serviceCollection.AddMemoryCache();
+            serviceCollection.AddAuthPolicies();
+        }
+        
+        private static void AddAuthPolicies(this IServiceCollection serviceCollection)
+        {
+            serviceCollection.AddAuthorization(AddPolicies);
+            serviceCollection.AddMvc(AddMvcPolicies);
+        }
+        
+        private static void AddPolicies(AuthorizationOptions options)
+        {
+            var allRoles = new []
+            {
+                AppRoles.CitizenRole, AppRoles.JudgeRole, AppRoles.RepresentativeRole, AppRoles.CaseAdminRole,
+                AppRoles.VhOfficerRole
```

Comment:
Doesn't the Panel Member role have a different user journey in video Web? Is it worth adding that?